snap/snapcraft.yaml

```diff
@@ -9,10 +9,7 @@ description: |
 # This software is unlikely to be used outside of desktop machines.
 # Let's limit the number of target architectures to reduce the build times.
 architectures:
-  # See https://forum.snapcraft.io/t/how-can-i-publish-a-snap-that-can-provide-binaries-for-all-the-supported-architectures/8556/4
-  # on why not just `build-on: [ amd64, i386 ]`.
-  - build-on: amd64
-  - build-on: i386
+  - build-on: [ amd64, i386 ]
```

Comment:
Hmmm is this change intentional? 🤔 the linked forum thread indicates that the old form is not equivalent to the new form

Reply:
Yeah, i read the linked forum but as far as i remember the old settings only kept building for one architecture and not for both, but just to be sure i will test it again for both options.

Reply:
I tested it again to make sure and it does not work as before, it only builds `amd64` architecture, not both. https://github.com/VirtusLab/git-machete/pull/489

Reply:
Here are the logs: 
1:
  - build-on: amd64
  - build-on: i386
  https://app.circleci.com/pipelines/github/VirtusLab/git-machete/1832/workflows/6fcbabeb-cc96-4e1d-9439-0da366195ca2/jobs/8169
2:
  - build-on: amd64
  - build-on: i386
https://app.circleci.com/pipelines/github/VirtusLab/git-machete/1831/workflows/41501906-deea-4143-b715-6b5c25a6b938/jobs/8166
3:
  - build-on: [ amd64, i386 ]
  https://app.circleci.com/pipelines/github/VirtusLab/git-machete/1833/workflows/a1663a60-9127-464f-a026-8ef05e9a0a55/jobs/8179

Reply:
Okay! good spot, now I see, it's indeed a single `..._multi.snap` package published (which I suppose contains binaries for both `amd64` and `i386`) instead of two separate `..._amd64.snap` and `_i386.snap` packages. Actually... support for `i386`  (i.e. 32-bit systems) has a minuscule significance nowadays anyway :)

Reply:
Let' leave as is then